src/main/java/edu/harvard/iq/dataverse/authorization/groups/GroupProvider.java

```diff
@@ -46,6 +38,18 @@
      */
     public Set<T> groupsFor( RoleAssignee ra, DvObject dvo );
```

Comment:
Can This go away now?

Reply:
No. It's in use by the GroupServiceBean.